data/remnant/remnant ships.txt

```diff
@@ -1183,33 +1188,30 @@ ship "Peregrine"
 		licenses
 			"Remnant Capital"
 		"cost" 41627000
-		"shields" 16400
-		"hull" 10600
+		"shields" 15400
+		"hull" 8600
 		"required crew" 9
 		"bunks" 25
 		"mass" 350
 		"drag" 4.3
 		"heat dissipation" 0.85
-		"fuel capacity" 1100
+		"fuel capacity" 900
 		"ramscoop" 3.25
-		"cargo space" 59
+		"cargo space" 49
 		"outfit space" 552
 		"weapon capacity" 220
 		"engine capacity" 171
-		"shield generation" 4.9
+		"shield generation" 3.9
 		"shield energy" 3.0
-		"hull repair rate" 2.5
+		"hull repair rate" 1.5
 		"hull energy" 0.90
-		"cloak" .04
-		"cloaking energy" 2.5
+		"cloak" .03
+		"cloaking energy" 3
 		"cloaking fuel" .05
 		"gaslining" 1
 		"outfit scan power" 18
 		"outfit scan efficiency" 22
-		"tactical scan power" 50
-		"ion protection" 0.25
-		"scramble protection" 0.25
-		"slowing protection" 0.25
+		"tactical scan power" 39
```

Comment:
```suggestion
		"tactical scan power" 50
```
The tactical scan is a tiny contributor to the excessive integrated outfit space, so there isn't a particularly strong need to nerf it.

Reply:
> scrambling resistance?

Good catch.

```suggestion
		"tactical scan power" 50
		"ion resistance" .15
		"scrambling resistance" .15
		"slowing resistance" .1
```

Reply:
Alternatively, if you want to preserve prior performance, but with resistances instead of protections for consistency with other Remnant content / differentiation from the Heliarchs' protections, then .31 ion/scrambling resistance works out the same on average as the .25 protection it had when in an Ember Waste ion storm.